test/fuzz/fuzzing.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+chmod u=rwx ./test/fuzz/fuzzing.sh
+rm -rf ./test/fuzz/output
+files=$(grep -r --include='**_fuzzer.go' --files-with-matches 'func Fuzz' .)
+mkdir -p ./test/fuzz/output
+#Adding package
+go get github.com/mdempsky/go114-fuzz-build
+go mod tidy
```

Comment:
We should use `go install` when the goal is to get the binary. Then there is no need for `go mod tidy` either.
```suggestion
go install github.com/mdempsky/go114-fuzz-build@latest
```

Reply:
updated this with the newer package.